### PR TITLE
animate send buttons on mobile

### PR DESCRIPTION
**Pull request by chrisnojima**
Animates send/actions in chat input

### CODE REVIEW

File: shared/chat/conversation/input-area/normal/platform-input.native.tsx [hunk 1]
```diff
@@ -211,39 +212,94 @@ class _PlatformInput extends PureComponent<PlatformInputPropsInternal, State> {
...
+    const hasValue = React.useRef(new Kb.NativeAnimated.Value(hasText ? 1 : 0)).current
+    const prevHasText = Container.usePrevious(hasText)
+
+    React.useEffect(() => {
+      Kb.NativeAnimated.timing(hasValue, {
+        duration: 200,
+        toValue: hasText ? 1 : 0,
+        useNativeDriver: true,
+      }).start()
+    }, [hasText, prevHasText, hasValue])
```
Comment (songgao) [inline shared/chat/conversation/input-area/normal/platform-input.native.tsx]: do we want to call `stop()` in cleanup?

Comment (songgao) [reply shared/chat/conversation/input-area/normal/platform-input.native.tsx]: Also why does the `prevHasText` matter here? It's just the the value we are changing into that matters right?

Comment (chrisnojima) [reply shared/chat/conversation/input-area/normal/platform-input.native.tsx]: fixed

Comment (chrisnojima) [reply shared/chat/conversation/input-area/normal/platform-input.native.tsx]: i don't think we need to call stop, i think its benign to let it run

**Pull request merged**